test/DAREditor/DAREditor.csproj

```diff
@@ -4,6 +4,18 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net472</TargetFramework>
     <UseWPF>true</UseWPF>
+    <LangVersion>10.0</LangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
>13.0.1 [](http://example.com/codeflow?start=57&length=6)

Can you change this to `$(Newtonsoft_Json_Version)`? To make this work, before line 3, add: `<Import Project="..\..\build\debuggertesting.settings.targets" />`